spec.emu

```diff
@@ -4,36 +4,289 @@
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
 <script src="./spec.js"></script>
 <pre class="metadata">
-title: Proposal Title Goes Here
-stage: -1
-contributors: Your Name(s) Here
+title: Measure
+status: proposal
+stage: 1
+contributors: Ben Allen, Jesse Alama
+location: https://github.com/tc39/proposal-measure/
 </pre>
 
-<emu-clause id="sec-demo-clause">
-  <h1>This is an emu-clause</h1>
-  <p>This is an algorithm:</p>
-  <emu-alg>
-    1. Let _proposal_ be *undefined*.
-    1. If IsAccepted(_proposal_) is *true*, then
-      1. Let _stage_ be *0*<sub>ℤ</sub>.
-    1. Else,
-      1. Let _stage_ be *-1*<sub>ℤ</sub>.
-    1. Return ? ToString(_stage_).
-  </emu-alg>
+<emu-intro id="sec-decimal-intro-">
+  <h1>Introduction</h1>
+  <p>This specification consists of two parts:</p>
+  <ul>
+    <li>The specification of the <a href="https://github.com/tc39/proposal-measure/" title="Measure proposal (GitHub)">Measure proposal</a> and everything related to it, proposed to be added to ECMA-262 in new sections;</li>
+    <li>A list of amendments to be made to ECMA-402.</li>
+  </ul>
+</emu-intro>
+
+<emu-clause id="sec-the-amount-object">
+  <h1>The Amount Object</h1>
+  <emu-intro id="sec-amount-intro">
+    <h1>Introduction</h1>
+    <p>An Amount is an object that wraps a digit String, representing a mathematical value, and a precision. One can intuitively understand an Amount as a number that, so to speak, knows its own precision.</p>
+  </emu-intro>
+
+  <emu-clause id="sec-amount-abstract-operations">
+    <h1>Abstract Operations</h1>
+
+    <emu-clause id="sec-amount-countsignificantdigits" type="abstract operation">
+      <h1>CountSignificantDigits(
+        _s_: a digit String
+      ): a positive integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It computes the number of significant digits in a given String, which is assumed to be a digit string in either decimal or exponential notation.</dd>
+      </dl>
+      <emu-alg>
+        1. <mark>Return 42.</mark>
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-amount-countfractiondigits" type="abstract operation">
+      <h1>CountFractionDigits(
+        _s_: a digit String
```

Comment:
and here, if it can have a `.` then "digit string" is a confusing name, and if not, then couldn't this AO just be "the length of _s_"?

Reply:
I've added a definition for this now.